ThirdParty/onepassword-app-extension/OnePasswordExtension.xcodeproj/project.pbxproj

```diff
@@ -14,6 +14,7 @@
 /* End PBXBuildFile section */
 
 /* Begin PBXFileReference section */
+		4484BAF62654227D0099A205 /* tr */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = tr; path = tr.lproj/OnePasswordExtension.strings; sourceTree = "<group>"; };
```

Comment:
Are we going to manually translate the 1P extension ourselves? Otherwise maybe we should remove Turkish from its localization (and the file it created below)

Reply:
@kylehickinson  Previous translations PR added those 1P extension strings I believe,
we would have to exclude them from transifex export/import

https://github.com/brave/brave-ios/pull/3689/files#diff-02c247007c81f9e7fe1fb203f7c49f20acd6c4bbb7bd072821ea5dbccc731b16

Do you think we should disable it somehow

Reply:
Interesting… They didnt exist before 1.26 translations, I wonder why all of a sudden they were pushed to transifex… I suppose at this point we may as well just localize it for Turkish as well 🤷‍♂️

Reply:
per offline discussion, we decided to leave translation, we have them already and they should not change, no cost for us